fusequery/query/src/configs/config.rs

```diff
@@ -27,8 +29,19 @@ lazy_static! {
         ver
     };
 }
-
-#[derive(Clone, Debug, serde::Deserialize, PartialEq, StructOpt, StructOptToml)]
+impl fmt::Debug for Config{
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{{ log_level: \"{}\", log_dir: \"{}\", num_cpus: {}, mysql_handler_host: \"{}\",
```

Comment:
It looks like a JSON. maybe we can consider the following two implementations:
1. Reference https://github.com/rust-lang/rust/blob/master/library/core/src/fmt/mod.rs#L523 to implemented.
2. Use serde::Serialize and [skip field attribute](https://serde.rs/field-attrs.html#skip)

Reply:
I will follow up. Thx.

Reply:
Or we can implement the following code and change the type of store_api_password type to Password
``` rust
struct Password {
    plain_text : String
}

impl Debug for Password {
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
        write!(f, "******");
    }
}
```

Reply:
.